### PR TITLE
added a naive voting service (no controller so far) - still WIP

### PR DESCRIPTION
**Pull request by themeshter**
Added the services layer for the voting option.

### CODE REVIEW

File: src/main/java/com/code4ro/legalconsultation/model/persistence/Vote.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "votes")
+@Getter
+@Setter
+public class Vote extends BaseEntity {
+  @Column(name = "vote_up")
+  private Boolean voteUp;
```
Comment (georgebejan) [inline src/main/java/com/code4ro/legalconsultation/model/persistence/Vote.java]: why do we need both `voteUp` and `voteDown`? can't we use the same convention as in the dto?

Comment (themeshter) [reply src/main/java/com/code4ro/legalconsultation/model/persistence/Vote.java]: Totally right - I don't know what got in my head at the moment of writing the code. One field is good enough to cover the three scenarios:
- up
- down
- vote abstaining (null).

I think I was just thinking "columnarly" so to speak (as if the upvotes will be counted on one column the the downvotes on another)...

This upcoming weekend I will redo it!

**Comment by RaduCStefanescu** [comment]
@themeshter could you please update this PR with the requested changes?

**Comment by themeshter** [comment]
Team, I apologize but I got sidetracked with office work (including a job rotation) and some personal activities. I am truly sorry for the silence and for letting you down! Is it possible for somebody else to pick up the code and continue it?

**Comment by georgebejan** [comment]
I will continue with this

**Pull request closed without merging**